Makefile

```diff
@@ -1,3 +1,31 @@
+MAKEFLAGS += --silent
+help:
+	@echo "Dont Fear the Makefile"
+	@echo "*DOCKER DEVELOPMENT*:"
+	@echo "make cluster - will start everything for you, but if there are no local images"
+	@echo "for Watcher and Child chain tagged with latest they will get pulled from our repository."
+	@echo "If you want to use your own image containers for Watcher and Child Chain"
+	@echo "use make docker-watcher && make docker-child_chain."
+	@echo "For rapid development that replaces containers with your code changes"
+	@echo "one can use make watcher-update or make child_chain-update."
+	@echo "BARE METAL DEVELOPMENT:"
+	@echo "ATTENTION ATTENTION ATTENTION"
+	@echo "-----------------------------"
+	@echo "You need to export environment variables."
+	@$(MAKE) -s raw-requirements
```

Comment:
Just tried running `make` after exporting the env variables, and I'm getting:

```
make[1]: *** No rule to make target `raw-requirements'.  Stop.
make: *** [help] Error 2
```

Is there something wrong with my `make`?

Reply:
oh, that rule got deleted! tnx